statsmodels/tsa/filters/tests/test_filters.py

```diff
@@ -347,6 +347,213 @@ def test_hpfilter():
     dta = macrodata.load_pandas().data['realgdp'].values
     res = column_stack((hpfilter(dta, 1600)))
     assert_almost_equal(res, hpfilt_res, 6)
+    # Test one-sided Hodrick-Prescott Filter. Results taken from matlab.
+    hpfilt_onesided_res = array([[ 0.00000000e+00, 2.71034900e+03],
```

Comment:
Can you seperate the test results into is own CSV file, and then load the CSV?  I'm not a big fan of large sets of values being hard coded.